create_card.py

```diff
@@ -70,7 +70,10 @@ def create_card(deckpath):
         # Call vim, set the cursor below the "FRONT" header,
         # allow snippets for our new filetype, set the syntax highlighting
         # so that it supports latex highlighting.
-        call(['vim', '+9', '-c set filetype=anki_vim', '-c set syntax=tex',
+        pattern = r'/\v\%\n\zs^$'
```

Comment:
This needs to be a raw string literal otherwise python will incorrectly interpret the escapes.